app/src/organisms/Devices/PipetteCard/__tests__/PipetteOverflowMenu.test.tsx

```diff
@@ -95,6 +103,42 @@ describe('PipetteOverflowMenu', () => {
     expect(props.handleCalibrate).toHaveBeenCalled()
   })
 
+  it('should render calibrate pipette offset text when the calibration wizard feature flag is set and no calibration exists', () => {
+    props = {
+      pipetteSpecs: mockLeftProtoPipette.modelSpecs,
+      mount: LEFT,
+      handleChangePipette: jest.fn(),
+      handleCalibrate: jest.fn(),
+      handleAboutSlideout: jest.fn(),
+      handleSettingsSlideout: jest.fn(),
+      isPipetteCalibrated: false,
+    }
+    mockUseFeatureFlag.mockReturnValue(true)
+    const { getByRole } = render(props)
+    const calibrate = getByRole('button', { name: 'Calibrate pipette offset' })
+    fireEvent.click(calibrate)
+    expect(props.handleCalibrate).toHaveBeenCalled()
+  })
+
+  it('does not render recalibrate pipette offset text when the calibration wizard feature flag is set', () => {
+    props = {
+      pipetteSpecs: mockLeftProtoPipette.modelSpecs,
+      mount: LEFT,
+      handleChangePipette: jest.fn(),
+      handleCalibrate: jest.fn(),
+      handleAboutSlideout: jest.fn(),
+      handleSettingsSlideout: jest.fn(),
+      isPipetteCalibrated: true,
+    }
```

Comment:
maybe, (others are the same as the props beforeEach()?)

```suggestion
    props.isPipetteCalibrated=true
```